Not able to connect skill

### #Typescript 

After recreating a fresh Skill and Virtual Assistant and connecting my Skill to my Virtual Assitant,
trying to launch the virtual Assistant fail with this error:

```
src/index.ts:50:7 - error TS2322: Type '{ "id": string; "name": string; "description": string; "iconUrl": string; "authenticationConnections": never[]; "actions": { "id": string; "definition": { "description": string; "slots": never[]; "triggers": { "utteranceSources": { ...; }[]; }; }; }[]; "msAppId": string; "endpoint": string; }[]' is not assignable to type 'ISkillManifest[]'.
  Type '{ "id": string; "name": string; "description": string; "iconUrl": string; "authenticationConnections": never[]; "actions": { "id": string; "definition": { "description": string; "slots": never[]; "triggers": { "utteranceSources": { ...; }[]; }; }; }[]; "msAppId": string; "endpoint": string; }' is not assignable to type 'ISkillManifest'.
    Types of property 'actions' are incompatible.
      Type '{ "id": string; "definition": { "description": string; "slots": never[]; "triggers": { "utteranceSources": { "locale": string; "source": string[]; }[]; }; }; }[]' is not assignable to type 'IAction[]'.
        Type '{ "id": string; "definition": { "description": string; "slots": never[]; "triggers": { "utteranceSources": { "locale": string; "source": string[]; }[]; }; }; }' is not assignable to type 'IAction'.
          Types of property 'definition' are incompatible.
            Type '{ "description": string; "slots": never[]; "triggers": { "utteranceSources": { "locale": string; "source": string[]; }[]; }; }' is not assignable to type 'IActionDefinition'.
              Types of property 'triggers' are incompatible.
                Type '{ "utteranceSources": { "locale": string; "source": string[]; }[]; }' is missing the following properties from type 'ITriggers': utterances, events

50 const skills: ISkillManifest[] = skillsRaw;
```


The skills.json seems fine (replaced my infos with *): 

`{
    "skills": [
        {
            "id": "*",
            "name": "*",
            "description": "This is the description of the *",
            "iconUrl": "http://*.azurewebsites.net/",
            "authenticationConnections": [],
            "actions": [
                {
                    "id": "*_Sample",
                    "definition": {
                        "description": "Trigger *",
                        "slots": [],
                        "triggers": {
                            "utteranceSources": [
                                {
                                    "locale": "en",
                                    "source": [
                                        "*#Sample"
                                    ]
                                }
                            ]
                        }
                    }
                }
            ],
            "msAppId": "*******************",
            "endpoint": "http://*.azurewebsites.net/api/skill/messages"
        }
    ]
}`

--- a/lib/typescript/botbuilder-skills/src/models/skillManifest.ts
+++ b/lib/typescript/botbuilder-skills/src/models/skillManifest.ts
@@ -45,9 +45,9 @@ export interface IActionDefinition {
  * Definition of the triggers for a given action within a Skill.
  */
 export interface ITriggers {
-    utterances: IUtterance[];
+    utterances?: IUtterance[];
     utteranceSources: IUtteranceSources[];
-    events: IEvent[];
+    events?: IEvent[];
 }
 
 /**

--- a/lib/typescript/botbuilder-skills/src/skillManifestGenerator.ts
+++ b/lib/typescript/botbuilder-skills/src/skillManifestGenerator.ts
@@ -134,6 +134,10 @@ export class SkillManifestGenerator {
                             });
                         });
 
+                        if (action.definition.triggers.utterances === undefined) {
+                            action.definition.triggers.utterances = [];
+                        }
+
                         action.definition.triggers.utterances.push({
                             locale: utteranceSource.locale,
                             text: utterancesToAdd

